Week-3/Excercise 3.4/Stack.js

```diff
@@ -0,0 +1,19 @@
+function createStack() {
+  // pulled out items from the object that was returned to avoid the access to items
+  let items = [];
```

Comment:
can be const?

Reply:
@anilrao316 Yes it can be const as well, but I have used let since that's what it was in the excercise.